### ISSUE TITLE
Teams Meetings - Customize your meeting invitations

### ISSUE THREAD

**Issue by NGOmarty** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#4681]
The title is a little misleading, as you not actually able to directly edit email greetings. We would like to edit the text of meeting invitations. At the moment, our dial-in number is incorrect and we would like to change it.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 470478ed-3b4e-ced8-0e28-6cda545b8400
* Version Independent ID: 95b4044b-3a47-a511-c726-b5ee75863441
* Content: [Manage meeting settings - Microsoft Teams](https://docs.microsoft.com/en-US/microsoftteams/meeting-settings-in-teams?WT.mc_id=TeamsAdminCenterCSH#feedback)
* Content Source: [Teams/meeting-settings-in-teams.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/meeting-settings-in-teams.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by scanum** [comment]
@NGOmarty Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by get-itips** [comment]
Hello @NGOmarty How are you?
As here we are limited to documentation changes, I need to kindly redirect you to other resource, which is [Microsoft Teams UserVoice](https://microsoftteams.uservoice.com) to make suggestions on changes to the application.
I searched, and I found [a suggestion](https://microsoftteams.uservoice.com/forums/555103-public/suggestions/33745942-meeting-invitation-customization) that matches your request, and I voted for it.
Hope this information helps you!
